### PR TITLE
Variable root location

### PR DESCRIPTION
**Pull request by dorpvom**
Added support for variable root location (e.g. `/radare` instead of `/`).

This is achieved by using `self.location.pathname` and stripping the filename (and the view prefix where necessary).

### CODE REVIEW

**Comment by dorpvom** [comment]
I have tested this by moving the contents of the `dev` and `dist` directories to my local radare2 sources. The `make run*` tests don't work on my machine (neither on master nor my own branch). Tips for testing would be appreciated.

File: www/lib/r2.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ var next_lastoff = 0;
 var prev_curoff = 0;
 var prev_lastoff = 0;
 var hascmd = false;
+var r2_root = self.location.pathname.split("/").slice(0, -2).join("/");
```
Comment (radare) [inline www/lib/r2.js]: use single quotes for strings

File: www/m/workers/disasmNavProvider.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 'use strict';
-importScripts('/m/r2.js');
+
+var m_root = self.location.pathname.split("/").slice(0, -1).join("/");
```
Comment (radare) [inline www/m/workers/disasmNavProvider.js]: use const and single quotes

**Comment by dorpvom** [comment]
Are there further change requests?

**Comment by radare** [comment]
Nope. Thanks!

**Pull request merged**